# If true, display a range of layers
# If false, display the whole trajectory
bool display_range_of_layers

# First layer in the range
uint32 first_layer

# Last layer in the range
uint32 last_layer
---
# If empty, service succeeded
# If failure, contains an error string
string error